### DISCUSSION TITLE
Configuration history page

### DISCUSSION THREAD

**Discussion by friebsch** [dainst/idai-field-client#167] [Ideas]
Changes to the configuration can have a major impact on data entry.

Since the configuration can be changed by all staff members and changes currently have to be accepted without further information, it would be important for transparency and data quality that changes made to the configuration can also be viewed afterwards.

An overview page with all the changes made and the persons responsible for them would be a good solution.

This expands issue 166

#166

**Comment by lsteinmann** [comment]
When designing the component for #223 it would be best to keep this plan in mind already, though it will have to be displayed in a separate way inside the project configuration tool - and will be a different issue.
